src/PhpSpec/Console/ContainerAssembler.php

```diff
@@ -390,8 +391,26 @@ private function setupLocator(IndexedServiceContainer $container)
         });
 
         $container->addConfigurator(function (IndexedServiceContainer $c) {
-            $suites = $c->getParam('suites', array('main' => ''));
+            $suites = [];
+            $namespaceProviderFactory = new Factory();
```

Comment:
ATM there is only one namespace provider... tell me if you fell this is overkill, I will just remove the factory and simplify the configuration to something less forward compatible, like `composer_namespace_provider: ~`

Reply:
please simplify it - other extensions can register new providers but they can provide their own configuration to do so